### DISCUSSION TITLE
Show hitboxes on death

### DISCUSSION THREAD

**Discussion by Entrap-ta** [Prevter/GDOpenHack#87] [Ideas]
When you die, the hitboxes should show so you can see what you died to

**Comment by dckrell** [comment]
its in the works, meanwhile if you want the latest beta (contains hitboxes on death) go to actions, if ur vanilla then go for the "build vanilla mod", if ur on geode go for the "build geode mod", for vanilla you just install it lmao, for geode open the gd folder find the geode folder replace the old mod file (settings carry over) don't blame anyone if it breaks btw
